web/src/components/map/menu.css

```diff
@@ -2,3 +2,50 @@
   position: absolute;
   bottom: 12px;
 }
+
+/* include "button" selector so other month dates are excluded
+ * (.clndr-cell-other-month seems not to be working; those spans
+ * don't have that class)
+ */
+button.clndr-cell {
+  background-color: lightblue;
+  border-radius: 50%;
+  opacity: 100%;
+}
+
+button.clndr-cell:hover {
+  background-color: azure;
+}
+
+/* include "button" and "disabled" selectors to override "hover" above */
+button.clndr-cell-disabled:disabled {
+  background-color: white;
+  color: black;
+  border-radius: 0;
+}
+
+/* include "button" selector to override above */
+button.clndr-cell-selected {
+  background-color: royalblue;
+  color: white;
+}
+
+.clndr-days {
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid white; /* hack to get rid of border */
+}
+
+/* Report tonight:
```

Comment:
Any outstanding questions we can move into follow-on issues and discussions. For now, I think we can remove this report and call the code good to merge.

Reply:
Whoops, that wasn't supposed to be there. Thanks.